### ISSUE TITLE
big-endian file parse failed

### ISSUE THREAD

**Issue by nobonobo** [TheBiggerGuy/pypcappy#1]
```
Traceback (most recent call last):
  File "sample.py", line 6, in <module>
    for packet in pcapng.packets:
  File "/run/media/irieda/AntergosRoot/home/irieda/Projects/humming-top/mg-replace/ht-mg-test/test/venv/lib/python3.5/site-packages/pypcappy/pcapNgFile.py", line 65, in <genexpr>
    self._packets = (block.packet for block in blocks if isinstance(
  File "/run/media/irieda/AntergosRoot/home/irieda/Projects/humming-top/mg-replace/ht-mg-test/test/venv/lib/python3.5/site-packages/pypcappy/pcapNgFile.py", line 29, in __next__
    self._peek_and_set_current_byteorder()
  File "/run/media/irieda/AntergosRoot/home/irieda/Projects/humming-top/mg-replace/ht-mg-test/test/venv/lib/python3.5/site-packages/pypcappy/pcapNgFile.py", line 56, in _peek_and_set_current_byteorder
    raise Exception('Invalid SectionHeaderBlock Byte-order Magic')
Exception: Invalid SectionHeaderBlock Byte-order Magic
```

https://github.com/TheBiggerGuy/pypcappy/blob/59db24299ae1289a5db57c5d5df83d6487262576/pypcappy/pcapNgFile.py#L43

`s/byteorder_little/byteorder_big/`

**Comment by TheBiggerGuy** [comment]
Fix in commit: 86e1c2c874c7f0818cec7931f86e15de9d262a74

**Comment by TheBiggerGuy** [comment]
Try v0.0.5

I am not currently working on this code. Is the basic function working for you apart from that bug?
